.gitmodules

```diff
@@ -46,4 +46,6 @@
 [submodule "elasticsearch/elasticpress"]
 	path = elasticsearch/elasticpress
 	url = https://github.com/Automattic/ElasticPress.git
-	branch = develop
```

Comment:
@chrean Looks like this PR is also changing the branch of the EP submodule?

Reply:
Yeah we should change the branch to `master` as discussed with 10up. Our production environment should point to `master` which is the release branch.
We can make this change in a separate PR anyway, does not need to happen here.

Reply:
That would be better, so we can more easily detect / test any changes between `develop` and `master` in that submodule, independently.

Reply:
Done here: 5664f5b9

Based on yesterday's conversation on Slack, I think this PR can be shipped now?

Reply:
Follow-up PR here: https://github.com/Automattic/vip-go-mu-plugins/pull/1460

Reply:
> I think this PR can be shipped now

I found a problem - it's not actually loading correctly because the filters aren't yet setup (b/c this is loading before we setup debug bar stuff). It was working in your testing b/c that test site had the plugin installed separately. Fixed in https://github.com/Automattic/vip-go-mu-plugins/pull/1450/commits/e78ed1482a9c239afbdfcbe1c66458d316e55944 (and I'm going to add some tests).